### DISCUSSION TITLE
Font class sizes stopped working..

### DISCUSSION THREAD

**Discussion by laurenceokite** [tailwindlabs/tailwindcss#14130] [Help] [answered]
text-sm, text-tiny and friends have stopped working in my project for some reason. All other classes are working as expected. Additionally, when added in @apply,  vite throws a postcss error that text-sm or any other text size class does not exist, even though the class is in my source files. 

*in tailwind.config.js*
`content: ["./src/**/*.{html,js,svelte,ts}"],`
- version: 3.3.6

**Comment by wongjn** [comment, marked as answer]
Consider checking any Tailwind configuration that may have removed the default value keys for `fontSize`. This can happen if you add custom configuration to `theme.fontSize` instead of `theme.extend.fontSize`. [See the documentation for details](https://tailwindcss.com/docs/theme#customizing-the-default-theme).

Otherwise, consider providing a project that reproduces the unexpected behavior.

**Reply by wongjn** [reply]
> https://github.com/laurenceokite/crossword-client-web
>
> nothing fancy in the tailwind config

Yep, so as per my previous comment, [you have custom configuration in `theme.fontSize`](https://github.com/laurenceokite/crossword-client-web/blob/b2854192e3341cfd75cb9a94d1fb41bc29c536cc/tailwind.config.cjs#L7) which eliminates Tailwind's defaults. Consider defining your custom configuration in `theme.extend.fontSize` instead.
